nifty-examples/src/test/java/com/facebook/nifty/server/TestNiftyOpenSslServer.java

```diff
@@ -262,6 +461,67 @@ public void testAttachTransportToUpdater() throws InterruptedException {
         Assert.assertNull(configUpdater.attachedTransport);
     }
 
+    @Test
+    public void testRotateTicketSeedFile() throws InterruptedException, IOException {
+        startServer();
+        OpenSslServerConfiguration config = (OpenSslServerConfiguration) server.getSSLConfiguration();
+
+        List<SessionTicketKey> actual = ImmutableList.copyOf(config.ticketKeys);
+        List<SessionTicketKey> expected = new TicketSeedFileParser().parse(getTicketSeedFile());
+        assertTicketKeysEqual(actual, expected);
+
+        // Rotate the ticket seeds file
+        updateTicketSeedFile(getResourceFileContents(Plain.class, "/ticket_seeds2.json"));
+        Thread.sleep(300); // Wait for three update cycles to make sure new configuration is in place
```

Comment:
I could add some stat counter APIs that return number of times the poller ran, and also number of times it found file changes.